iocore/cache/CacheHosting.cc

```diff
@@ -592,23 +595,27 @@ void
 ConfigVolumes::read_config_file()
 {
   ats_scoped_str config_path;
-  char *file_buf;
 
   config_path = RecConfigReadConfigPath("proxy.config.cache.volume_filename");
   ink_release_assert(config_path);
 
   Note("%s loading ...", ts::filename::VOLUME);
 
-  file_buf = readIntoBuffer(config_path, "[CacheVolition]", nullptr);
-  if (file_buf == nullptr) {
-    Error("%s failed to load", ts::filename::VOLUME);
-    Warning("Cannot read the config file: %s", (const char *)config_path);
-    return;
-  }
+  std::error_code ec;
+  std::string content{ts::file::load(ts::file::path{config_path}, ec)};
 
-  BuildListFromString(config_path, file_buf);
-  ats_free(file_buf);
+  if (ec) {
+    switch (ec.value()) {
+    case ENOENT:
+      Warning("Cannot open the config file: %s - %s", (const char *)config_path, strerror(ec.value()));
+      break;
+    default:
+      Error("%s failed to load: %s", ts::filename::VOLUME, strerror(ec.value()));
```

Comment:
Why use `VOLUME` instead of `config_path` for consistency between the two cases?

Reply:
yup, done!